pybleau/app/model/dataframe_analyzer.py

```diff
@@ -199,6 +202,15 @@ def shuffle_filtered_df(self):
 
     # Traits Listeners --------------------------------------------------------
 
+    @observe("column_descr:items")
+    def check_col_descr_change(self, event):
+        for key, value in event.added.items():
+            if key not in self.source_df.columns:
+                msg = f"Note added for invalid column {key}. Removing it... " \
+                    f"(Note was: {value})"
+                logger.error(msg)
+                self.column_descr.pop(key)
```

Comment:
As referenced in the tests for this feature, it may be worth addressing when a column description is deleted.

Reply:
Yes.